anybox/recipe/odoo/base.py

```diff
@@ -218,15 +218,19 @@ def __init__(self, buildout, name, options):
         self.vcs_clear_locks = clear_locks == 'true'
         clear_retry = options.get('vcs-clear-retry', '').lower()
         self.clear_retry = clear_retry == 'true'
-
-        if self.bool_opt_get(WITH_ODOO_REQUIREMENTS_FILE_OPTION):
-            logger.debug("%s option: adding 'pip' to the recipe requirements",
-                         WITH_ODOO_REQUIREMENTS_FILE_OPTION)
```

Comment:
Let's keep this!

Reply:
Restored

Reply:
Oh, you have restored this but in an additional commit instead of in the rebase commit. Can you move it back?

Reply:
Perfect, thanks!